### DISCUSSION TITLE
Code Coverage

### DISCUSSION THREAD

**Discussion by warengonzaga** [web3phl/directory#10] [Brainstorm]
Just wanna mention this here, and move this once this is a thing with the development. Feel free to discuss here anything related to code coverage for the project. Let me know your thoughts!

So far, still thinking if we are going to use Jest or something else.
Suggest below! ✌🔥

**Comment by jnebab** [comment]
for unit testing, Jest is still probably the best. And as far as I understood the requirements and scope of the project from the meeting, the coverage is small, and might mostly focus on filter, search, pagination, and maybe bookmarking if we ever consider it.

**Reply by warengonzaga** [reply]
Do you think we need a smaller unit test for our setup?
